### PR TITLE
CORCI-668 build: Add Ubuntu packaging

### PR DESCRIPTION
**Pull request by JohnMalmberg**
New Ubuntu dockerfile and debian files.

Signed-off-by: John.E.Malmberg <john.e.malmberg@intel.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -166,12 +168,42 @@ pipeline {
                         }
                     }
                 }
+                // This stage will not be needed once Cart is updated
+                // to a newer scons_local
```
Comment (brianjmurrell) [inline Jenkinsfile:172]: Which stage is that referring to?

Comment (JohnMalmberg) [reply Jenkinsfile:172]: The Ubuntu stage.  Ubuntu 18.04 and later do not need this package except that scons_local is always checking for it to be present.  There is a ticket in progress to fix this.

Once that work is complete, the Ubuntu dockerfiles for those packages can be updated to not install this package and then all Ubuntu related stuff can be removed from this repository.

Comment (brianjmurrell) [reply Jenkinsfile:172]: Ahhh.  Maybe move the comment into the stage (i.e. after line 173) then?

**Review by brianjmurrell — APPROVED** [review]
Up to you whether to move the comment or not.

**Comment by JohnMalmberg** [comment]
Not moving the comment, as will be removing all the Ubuntu 18+ code in the future.

**Pull request merged**